gamemodes/attdef.pwn

```diff
@@ -9634,6 +9644,16 @@ public OnScriptUpdate()
   			PlayerTextDrawSetString(i, FPSPingPacket[i], sprintf("%sFPS ~r~%d %sPing ~r~%d %sPacketLoss ~r~%.1f%%", MAIN_TEXT_COLOUR, Player[i][FPS], MAIN_TEXT_COLOUR, GetPlayerPing(i), MAIN_TEXT_COLOUR, NetStats_PacketLossPercent(i)));
             Update3DTextLabelText(Player[i][InfoLabel], -1, sprintf("%sPL: {FFFFFF}%.1f%%\n%sPing: {FFFFFF}%d\n%sFPS: {FFFFFF}%d", TextColor[Player[i][Team]], NetStats_PacketLossPercent(i), TextColor[Player[i][Team]], GetPlayerPing(i), TextColor[Player[i][Team]], Player[i][FPS]));
 		}
+		
+		// Update Speedometer
```

Comment:
If it's for that purpose then just make a /vh command (show vehicle's hp percentage and damage status in a client message). It's short enough to type it quickly and you don't have to worry about toggling it on and off. I'm very sensitive to stuff under OnScriptUpdate and OnPlayerUpdate. You shouldn't add code there unless you're in a really bad need.